modules/core/keeper/msg_server_test.go

```diff
@@ -18,40 +15,11 @@ import (
 	ibcmock "github.com/cosmos/ibc-go/v3/testing/mock"
 )
 
-const height = 10
-
 var (
 	timeoutHeight = clienttypes.NewHeight(0, 10000)
 	maxSequence   = uint64(10)
 )
 
-type KeeperTestSuite struct {
-	suite.Suite
-
-	coordinator *ibctesting.Coordinator
-
-	chainA *ibctesting.TestChain
-	chainB *ibctesting.TestChain
-}
-
-// SetupTest creates a coordinator with 2 test chains.
-func (suite *KeeperTestSuite) SetupTest() {
```

Comment:
Why deleting this test?

Reply:
@yzang2019 this wasn't deleted. This setup `SetupTest()` was just moved to `modules/core/keeper/keeper_test.go` https://github.com/sei-protocol/sei-ibc-go/pull/34/files/629fca36c47370cfeb8a90a4d14588b95f0b5063#diff-e3cf65de9a7dcd0eed1e68d80c4f911722d0c9577f4c593b4a3565ae0172b695R29